crates/iota-indexer/src/types.rs

```diff
@@ -543,6 +564,64 @@ pub struct TxIndex {
     pub move_calls: Vec<(ObjectID, String, String)>,
 }
 
+#[cfg(feature = "pg_integration")]
+impl TxIndex {
+    const MAX_OBJECTS: usize = 1000;
```

Comment:
nit: Since these constants are only used within the `random()` function, should we maybe move them there? As they don't seem to have semantic meaning for `TxIndex` impl itself.

Reply:
fixed: https://github.com/iotaledger/iota/pull/7969/commits/97fe091bf8eb48280971c5c6ea5ce49ef2a5169c